.github/workflows/scout-pr.yml

```
@@ -0,0 +1,63 @@
+name: Container security (Docker Scout)
+
+on:
+  pull_request:
+    branches: [ main ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)